libraries/AP_Proximity/AP_Proximity_LeddarVu8.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include "AP_Proximity.h"
+#ifndef AP_PROXIMITY_LEDDARVU8_ENABLED
+#define AP_PROXIMITY_LEDDARVU8_ENABLED HAL_PROXIMITY_ENABLED
+#endif
+
+#if (HAL_PROXIMITY_ENABLED && AP_PROXIMITY_LEDDARVU8_ENABLED)
+#include "AP_Proximity_Backend_Serial.h"
+//#include "AP_Proximity_Backend.h"
+
+#define LEDDARVU8_PAYLOAD_LENGTH (8*2)
+#define LEDDARVU8_ADDR_DEFAULT              0x01    // modbus default device id
+#define LEDDARVU8_DIST_MAX_CM               18500   // maximum possible distance reported by lidar
+#define LEDDARVU8_DIST_MIN_CM               5       // maximum possible distance reported by lidar
+#define LEDDARVU8_OUT_OF_RANGE_ADD_CM       100     // add this many cm to out-of-range values
+#define LEDDARVU8_TIMEOUT_MS                200     // timeout in milliseconds if no distance messages received
+#define LEDDARVU8_START_ANGLE 		        0.0f  // hardcoded for 48 deg FOV: Starting 2-D horizontal angle of distances received in payload
+#define LEDDARVU8_ANGLE_STEP		        45.0f    // hardcoded for 48 deg FOV: Angle step size of each distance received. Starts from LEDDARVU8_START_ANGLE
+class AP_Proximity_LeddarVu8 : public AP_Proximity_Backend_Serial
+{
+
+public:
+
+    using AP_Proximity_Backend_Serial::AP_Proximity_Backend_Serial;
+
+    // update the state of the sensor
+    void update(void) override;
+
+    // get maximum and minimum distances (in meters) of sensor
+    float distance_max() const override { return LEDDARVU8_DIST_MAX_CM / 100; }
```

Comment:
Multiplications are faster on the hardware. Here it certainly just computes it of course at the compiler level of course. 
```suggestion
    float distance_max() const override { return LEDDARVU8_DIST_MAX_CM * 0.01; }
```

Reply:
fixed in latest push